dosomething-qa/main.tf

```diff
@@ -31,9 +31,9 @@ module "fastly-backend" {
   northstar_domain  = module.northstar.domain
   northstar_backend = module.northstar.backend
 
-  rogue_name    = module.rogue.name
-  rogue_domain  = module.rogue.domain
-  rogue_backend = module.rogue.backend
+  rogue_name    = "dosomething-rogue-qa"
+  rogue_domain  = "activity-qa.dosomething.org"
+  rogue_backend = "dosomething-rogue-qa.herokuapp.com"
```

Comment:
We had to remove these references in order to avoid errors on Fastly QA. We should perhaps see if we can find a way to make these two modules less tightly coupled.